### ISSUE TITLE
Updated subs-only version

### ISSUE THREAD

**Issue by JimmyWoooo** [TwitchRecover/TwitchRecover#221]
Hey everyone, before I begin I would to thank @ItIckeYd for providing the updated server links. I downloaded their python script last week and tested it out on a Sub-Only video and it worked but it's going to take a lot of time if you have a bunch of videos. I updated the Fuzz.java file to include the new links and it works again(tried it on my computer and a virtual machine with nothing on it). I've attached the new executable file below in the form of text since Github doesn't allow me to upload exe files. All you need to do is rename the extension back to exe. 

[TwitchRecover.txt](https://github.com/TwitchRecover/TwitchRecover/files/8656068/TwitchRecover.txt)

If it doesn't work for you then gg.

**Comment by ItIckeYd** [comment]
@JimmyWoooo good point! I’ll surround the main with a while loop based on user input if they would like to recover another vod for efficiency…also that’s awesome that you updated the code! Thank you

**Comment by GmorKodi** [comment]
@JimmyWoooo would it be possible for you to upload the JAR so I could run it on my mac?

**Comment by JimmyWoooo** [comment]
@ItIckeYd You were the real VIP 🎩

@GmorKodi Yea, sure. Just rename it.
[TwitchRecoverJar.txt](https://github.com/TwitchRecover/TwitchRecover/files/8657365/TwitchRecoverJar.txt)

**Comment by ujrarozu** [comment]
> @ItIckeYd You were the real VIP 🎩
> 
> @GmorKodi Yea, sure. Just rename it. [TwitchRecoverJar.txt](https://github.com/TwitchRecover/TwitchRecover/files/8657365/TwitchRecoverJar.txt)

bro can i contact u somewhere i have a question
